CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ option(IREE_BUILD_COMPILER "Builds the IREE compiler." ON)
 option(IREE_BUILD_TESTS "Builds IREE unit tests." ON)
 option(IREE_BUILD_SAMPLES "Builds IREE sample projects." ON)
 option(IREE_BUILD_DEBUGGER "Builds the IREE debugger app." OFF)
+option(IREE_BUILD_PYTHON_BINDINGS "Builds the IREE python bindings" OFF)
```

Comment:
Thanks for adding another option, that will definitely help build out support in this area while keeping the rest of the build stable.